build/appveyor.sh

```diff
@@ -3,17 +3,20 @@
 # Build script run by Appveyor. Might eventually be less
 # single-purpose, but let's get coverage going ASAP...
 
-cd $(dirname $0)/..
```

Comment:
(Not that it matters given that it's always actually going to be run from the root :)